lib/elastic_record/index.rb

```diff
@@ -65,6 +65,11 @@ def enable!
       @disabled = false
     end
 
+    def load_from_source!
+      @load_from_source = true
```

Comment:
Use `self.load_from_source`, in case the implementation does more than set a var?